src/ui/windows/main_window/store_page.rs

```diff
@@ -157,6 +157,8 @@ impl SimpleAsyncComponent for StorePage {
 
                 self.show_game_page = true;
             }
+
+            StorePageInput::Activate => {}
```

Comment:
Why did you return this event back when it does *nothing*?

Reply:
It doesn't do nothing, it runs the code below that which updates the back button showing when switching between the store and library.